config.py

```diff
@@ -44,6 +44,18 @@ class Config:
 
     VCAP_SERVICES = None
 
+    DM_FRAMEWORK_TO_ES_INDEX_MAPPING = {
```

Comment:
hmm I just spotted that this name is potentially confusing, because it uses the word 'mapping', which is - uh - overloaded now somewhat!

Perhaps `DM_INDEX_NAME_BY_FRAMEWORK_AND_DOC_TYPE`? (eww long I know)

I was going to suggest we come up with some more self-documenting structure like

```
DM_INDEXES = [
    { 'framework-slug': 'g-cloud-9', 'doc-type': 'services', 'index-name': 'g-cloud-9', }
    # etc
]

But then you'd have to have a function somewhere that parsed it out, and I've no idea where that would go, so let's not do that.

Reply:
How about just `DM_FRAMEWORK_TO_ES_INDEX`?

Reply:
Fine by me